libs/etterlatte-ktor/src/main/kotlin/route/RouteUtils.kt

```diff
@@ -200,6 +200,12 @@ suspend inline fun PipelineContext<*, ApplicationCall>.withParam(
 suspend inline fun PipelineContext<*, ApplicationCall>.hentNavidentFraToken(onSuccess: (navident: String) -> Unit) {
     val navident = call.firstValidTokenClaims()?.get("NAVident")?.toString()
     if (navident.isNullOrEmpty()) {
+        val bruker = call.brukerTokenInfo
```

Comment:
Hm skjønner behovet her, men slik jeg ser bruken av denne er den implisitt til å kun godta saksbehandler og ikke systembruker. feks i `kommerbarnettilgode` -> `Grunnlagsopplysning.Saksbehandler.create(navIdent),`
Disse må gjenspeile endringene før denne kan komme inn, evt ha en egen for de endepunktene du trenger den for.

Reply:
Ja det er nok mer fornuftig å bruke `brukerTokenInfo` direkte!